addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4314,6 +4332,10 @@ registry.ImageTools = SnippetOptionWidget.extend({
             return this.$target.attr('alt');
         } else if (methodName === 'setTitle') {
             return this.$target.attr('title');
+        } else if (methodName === 'transform') {
+            this.$el.find('[data-transform]').toggleClass('active', this._isTransformed());
+        } else if (methodName === 'crop') {
+            this.$el.find('[data-crop]').toggleClass('active', this._isCropped());
```

Comment:
`_computeWidgetState` is not in charge of updating the UI, just returning its status. And according to that status, `updateUI` is in charge of adding the `active` class. By doing this here, you are duplicating the code which adds the `active` class and you are mixing the concepts. Should be something like this:
```javascript
return isTransformed ? 'true' : '';
```
and the option method should be defined `data-transform="true"`